redis_rate_limit/__init__.py

```diff
@@ -103,21 +113,27 @@ def has_been_reached(self):
         """
         return self.get_usage() >= self._max_requests
 
-    def increment_usage(self):
+    def increment_usage(self, increment_by=1):
         """
         Calls a LUA script that should increment the resource usage by client.
 
         If the resource limit overflows the maximum number of requests, this
         method raises an Exception.
 
+        :param increment_by: The count to increment the rate limiter by.
+        This is typically 1, but higher or lower values are provided
```

Comment:
Yeah, the code doesn't currently check against negative values, but I could add one.

I think the more common scenario might be to increase the bucket size instead of decreasing the request number.

Reply:
Let's keep the behavior consistent with the function name and make this only increase.